packages/theming/atlas/src/theme/web/custom-variables.scss

```diff
@@ -545,6 +545,9 @@ $accordion-success-border-color: $btn-success-border-color;
 $accordion-warning-border-color: $btn-warning-border-color;
 $accordion-danger-border-color: $btn-danger-border-color;
 
+// Other
+$accordion-border-width: 1px;
```

Comment:
Does this needs to be a variable?? We can simply customize the css

Reply:
I added it as a variable, because it's used in many places: helper classes, calculation of border radius in widget styling. Another solution is to add the variable inside the widget styles file, so that it's also available in the helper styles file. Do you want me to do that?

Reply:
I would rather add in the widget and if users really want to change the border width they can override the styles. I think the amount of people changing this value will be really low.

Reply:
Done.